.github/dependabot.yml

```
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: "/"
+    schedule:
+      interval: monthly
+      time: "05:00"
+      timezone: "America/Montreal"
+    open-pull-requests-limit: 50
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: monthly
+      time: "05:00"
+      timezone: "America/Montreal"
```

```suggestion
```suggestion
  - package-ecosystem: npm
    directory: "/"
    schedule:
      interval: monthly
      time: "05:00"
      open-pull-requests-limit: 50
      timezone: "America/Montreal"
```
```